README.md

```diff
@@ -1,7 +1,45 @@
-# edge-sync-server
+# Edge Sync Server
 
-Questionss
-- best way to check if document exists, currently using head
-- lastGitHash, lastGitTime
-- schema validation library
+A HTTP API server to store, retrieve, and synchronize encrypted data between clients and servers. It uses CouchDB as the backend and Express as the HTTP server.
 
+## Usage
+
+### Install
+
+```
+yarn
+```
+
+### Configuration
+
+App configuration settings are managed by a `config.json`. The schema for this file is located in `src/config.schema.ts` and is written using [cleaners](https://www.npmjs.com/package/cleaners). You must create a new `config.json` file matching the expected config type from the schema file. The config file name can be controlled with the `CONFIG` envirnoment varariable.
```

Comment:
Please add a `config.sample.json` file. This is our convention versus telling the user to look at a cleaner.